src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -484,9 +484,7 @@ class TFAlbertMainLayer(tf.keras.layers.Layer):
     def __init__(self, config, **kwargs):
         super().__init__(**kwargs)
         self.num_hidden_layers = config.num_hidden_layers
-        self.output_attentions = config.output_attentions
-        self.output_hidden_states = config.output_hidden_states
-        self.return_dict = config.use_return_dict
+        self.config = config
```

Comment:
I always thought there was a reason why we never save the config in lower TF modules...but if there is not `self.config = config` is totally fine for me @LysandreJik

Reply:
Nothing prevent you to do this as long as you have a proper `get_config()` method, which is our case with the `keras_serializable` decorator. Afterward we can even think to remove all the `self.parameter = config.parameter`.

Reply:
If we can save the config instead of storing the attributes, it's way cleaner, so I'm all for it :-)